gems/pending/appliance_console/database_replication.rb

```diff
@@ -119,5 +116,14 @@ def run_repmgr_command(cmd, params = {})
       Process.wait(pid)
       $CHILD_STATUS.success?
     end
+
+    def primary_connection_hash
```

Comment:
@carbonin Looks good. I have one simple nit: I would like to see this method name changed to `connection_hash`. Since this will be used by both the primary and secondary.